controllers/twist_controller/src/twist_controller.cpp

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 Kristófi Mihály
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "twist_controller.hpp"
+
+namespace kuka_controllers
+{
+controller_interface::CallbackReturn TwistController::on_init()
+{
+  return controller_interface::CallbackReturn::SUCCESS;
+}
+
+controller_interface::InterfaceConfiguration TwistController::
+command_interface_configuration()
+const
+{
+  controller_interface::InterfaceConfiguration config;
+  config.type = controller_interface::interface_configuration_type::INDIVIDUAL;
+  // config.names.emplace_back("twist commands");
+  config.names.emplace_back("twist/linear/x");
```

Comment:
you should use the defined constexpressions here. the main reason for defining them is to be able to use them at more places